### ISSUE TITLE
[Bug] Cosmetic error only: Shows vague error message while printing

### ISSUE THREAD

**Issue by KenLucke** [cp2004/OctoPrint-EEPROM-Marlin#9]
**Describe the bug**
Cosmetic bug - error message is somewhat confusing.

**Steps To Reproduce**
Start a print
Open Editor tab shows this error, despite the editor working just fine before starting the print:
![CR-10 v2 (Yellow) LAN  OctoPrint  - 72% 2020-10-19 16-05-18](https://user-images.githubusercontent.com/57936294/96521292-c6015c80-1225-11eb-8444-13293b711529.jpg)
This could confuse someone into thinking that the plugin won't work for them or that there's been some sort of massive sudden EEPROM fault, when, in fact, it works fine if they were not printing at the moment.

Tiny little thing in the larger order of things, but I thought it was worth pointing out.

**Expected behavior**
Should say something on the order of "Error: Either this plugin does not function with your machine (only Marlin based EEPROM machines are supported) , or the plugin is currently disabled because a print is in progress."

**Logs**

`octoprint.log`:  not needed for this one.

**Versions, system information**
**Plugin**:
Version:  2.0
**OctoPrint**
Version: 1.4.2
Operating System running OctoPrint:
Mac OS X 10.15.7
Printer model & firmware version: n/a
Browser: n/a

**Additional context, screenshots, anything else you think might be useful**

**Comment by cp2004** [comment]
Looks like that's actually the two different error messages *at the same time*, they should be two different cases. Think this happens if you start a print, then load the UI later on. Will fix.

**Comment by cp2004** [comment]
Fixed for the next release. Error messages include:
Disconnected:
![image](https://user-images.githubusercontent.com/31997505/100636818-0aeed780-332a-11eb-8831-82ef4e500726.png)
Read-only mode:
![image](https://user-images.githubusercontent.com/31997505/100636653-d844df00-3329-11eb-9931-6201396d9a63.png)
Firmware not marlin:
![image](https://user-images.githubusercontent.com/31997505/100636916-2528b580-332a-11eb-81e9-8d8d3bb590a9.png)

And maybe more, I can't remember!

**Comment by cp2004** [comment]
3.0.0 has been released 🎉
